External/FEXCore/Source/Interface/Core/CPUID.cpp

```diff
@@ -109,6 +109,29 @@ FEXCore::CPUID::FunctionResults CPUIDEmu::Function_01h() {
   return Res;
 }
 
+// 2: Cache and TLB information
+FEXCore::CPUID::FunctionResults CPUIDEmu::Function_02h() {
+  FEXCore::CPUID::FunctionResults Res{};
+
+  Res.eax = 0x1 | // lowest byte always 0x01
```

Comment:
Well... the Intel manual actually goes into more detail about how this is the count for the you need to read this cpuid to get every single cache/tlb descriptor byte.

But I think all intel CPUs do actually fit everything inside the 15 bytes supplied here, so it's probably true that it's always 0x1.